docs/superpowers/plans/2026-05-09-true-client-server-mode.md

```
@@ -0,0 +1,543 @@
+AXON_SERVER_URL=http://127.0.0.1:8001 axon scrape https://example.com --json --max-pages 1
+AXON_SERVER_URL=http://127.0.0.1:8001 axon crawl status <job_id> --json
+```
+
+Run: `just client-server-smoke`
+
+Expected: server-mode calls hit the canonical server and outputs are server-owned.
+
+- [ ] **Step 4: Full gate**
+
+Run:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)